aviary/subsystems/geometry/gasp_based/test/test_wing.py

```diff
@@ -331,6 +331,10 @@ def test_case1(self):
 
 
 class WingGroupTestCase2(unittest.TestCase):
+    """
+    Wing with both folds and struts which has fold dimensional location and strut dimensional location specified
```

Comment:
add `with the fold at the strut connection`

Reply:
to differentiate it from case4

Reply:
done